TestForwardBasic is flaky

Seems to be consistently reproducible with:

```
go test ./pss/... -count=2
```

```
[33mWARN [0m[07-11|13:14:07.038] caught subscription error in pss sub topic 7f64614e: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:07.038] caught subscription error in pss sub topic 7f64614e: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:08.058] caught subscription error in pss sub topic 7f64614e: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:08.058] caught subscription error in pss sub topic 7f64614e: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:09.074] caught subscription error in pss sub topic 7f64614e: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:09.074] caught subscription error in pss sub topic 7f64614e: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:25.078] Can't stop node                          [33mid[0m=b2f3a69d52f98037 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:14:25.078] Can't stop node                          [33mid[0m=ece30c5ce40c2769 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:14:25.079] Can't stop node                          [33mid[0m=304c18a9ab1f0bc4 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:14:25.079] Can't stop node                          [33mid[0m=c7c080ce0b5ec6ed [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:14:25.080] Can't stop node                          [33mid[0m=0ffafe08e99119b2 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:14:25.080] Can't stop node                          [33mid[0m=9ed40b70e4c72997 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:14:25.081] Can't stop node                          [33mid[0m=d46598bfcf4437f8 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:14:25.081] Can't stop node                          [33mid[0m=6a517303172afeb0 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:14:25.081] Can't stop node                          [33mid[0m=689c9a0f57a83846 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:14:25.082] Can't stop node                          [33mid[0m=72d3d1448858e393 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:14:25.082] Can't stop node                          [33mid[0m=348e9351b10625a6 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:14:25.082] Can't stop node                          [33mid[0m=3a9a3b38d0dcd484 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:14:25.083] Can't stop node                          [33mid[0m=769c948d795d1319 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:14:25.083] Can't stop node                          [33mid[0m=0160436a4791d77b [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:14:25.083] Can't stop node                          [33mid[0m=55bcbf1392d0cd77 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:14:37.091] noproxhandler                            [33mcaller[0m=pss.go:492
[33mWARN [0m[07-11|13:14:37.091] noproxhandler                            [33mcaller[0m=pss.go:492
[33mWARN [0m[07-11|13:14:37.091] noproxhandler                            [33mcaller[0m=pss.go:492
[31mERROR[0m[07-11|13:14:37.097] peer doesn't have matching pss capabilities, skipping [31mpeer[0m=0x39b394d4dfa635c975bf2db4796fa61d440a78b09ce43476c008a02546bb813b [31mcaps[0m=[nopss/1] [31mcaller[0m=pss.go:689
[31mERROR[0m[07-11|13:14:37.097] peer doesn't have matching pss capabilities, skipping [31mpeer[0m=0xc5c527b1d65eafa5c22f3bd39823291e9c9eeb33342817616f4732df8f18ba28 [31mcaps[0m=[pss/0]   [31mcaller[0m=pss.go:689
[31mERROR[0m[07-11|13:14:37.097] peer doesn't have matching pss capabilities, skipping [31mpeer[0m=0x39b394d4dfa635c975bf2db4796fa61d440a78b09ce43476c008a02546bb813b [31mcaps[0m=[nopss/1] [31mcaller[0m=pss.go:689
[31mERROR[0m[07-11|13:14:37.097] peer doesn't have matching pss capabilities, skipping [31mpeer[0m=0xc5c527b1d65eafa5c22f3bd39823291e9c9eeb33342817616f4732df8f18ba28 [31mcaps[0m=[pss/0]   [31mcaller[0m=pss.go:689
[33mWARN [0m[07-11|13:14:37.098] norawhandler                             [33mcaller[0m=pss.go:488
[33mWARN [0m[07-11|13:14:37.104] Served pss_sendRaw                       [33mreqid[0m=3 [33mt[0m=9.543¬µs [33merr[0m="invalid message length" [33mcaller[0m=handler.go:300
[33mWARN [0m[07-11|13:14:37.104] Served pss_sendRaw                       [33mreqid[0m=4 [33mt[0m=9.786¬µs [33merr[0m="address too long"       [33mcaller[0m=handler.go:300
[33mWARN [0m[07-11|13:14:37.616] caught subscription error in pss sub topic deadbeef: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:37.616] caught subscription error in pss sub topic deadbeef: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:38.128] caught subscription error in pss sub topic deadbeef: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:38.128] caught subscription error in pss sub topic deadbeef: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:38.646] caught subscription error in pss sub topic deadbeef: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:38.646] caught subscription error in pss sub topic deadbeef: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:39.160] caught subscription error in pss sub topic 75d158a6: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:39.160] caught subscription error in pss sub topic 75d158a6: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:39.678] caught subscription error in pss sub topic 75d158a6: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:39.678] caught subscription error in pss sub topic 75d158a6: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:40.186] caught subscription error in pss sub topic 75d158a6: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:40.186] caught subscription error in pss sub topic 75d158a6: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:40.949] caught subscription error in pss sub topic 75d158a6: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:40.949] caught subscription error in pss sub topic 75d158a6: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:41.717] caught subscription error in pss sub topic 75d158a6: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:41.717] caught subscription error in pss sub topic 75d158a6: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:42.482] caught subscription error in pss sub topic 75d158a6: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:42.482] caught subscription error in pss sub topic 75d158a6: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:45.695] caught subscription error in pss sub topic 75d158a6: EOF [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:45.695] caught subscription error in pss sub topic 75d158a6: EOF [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:45.696] Can't stop node                          [33mid[0m=b2f3a69d52f98037 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:14:45.696] caught subscription error in pss sub topic 75d158a6: EOF [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:45.696] Can't stop node                          [33mid[0m=ece30c5ce40c2769 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:14:45.696] caught subscription error in pss sub topic 75d158a6: EOF [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:45.697] Can't stop node                          [33mid[0m=304c18a9ab1f0bc4 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:14:45.697] caught subscription error in pss sub topic 75d158a6: EOF [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:45.697] Can't stop node                          [33mid[0m=c7c080ce0b5ec6ed [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:14:45.697] caught subscription error in pss sub topic 75d158a6: EOF [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:45.697] Can't stop node                          [33mid[0m=0ffafe08e99119b2 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:14:45.697] caught subscription error in pss sub topic 75d158a6: EOF [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:45.697] Can't stop node                          [33mid[0m=9ed40b70e4c72997 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:14:45.698] caught subscription error in pss sub topic 75d158a6: EOF [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:45.698] Can't stop node                          [33mid[0m=d46598bfcf4437f8 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:14:45.698] caught subscription error in pss sub topic 75d158a6: EOF [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:45.698] Can't stop node                          [33mid[0m=6a517303172afeb0 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:14:45.698] caught subscription error in pss sub topic 75d158a6: EOF [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:45.698] Can't stop node                          [33mid[0m=689c9a0f57a83846 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:14:45.698] caught subscription error in pss sub topic 75d158a6: EOF [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:45.699] Can't stop node                          [33mid[0m=72d3d1448858e393 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:14:45.699] caught subscription error in pss sub topic 75d158a6: EOF [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:45.699] Can't stop node                          [33mid[0m=348e9351b10625a6 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:14:45.699] caught subscription error in pss sub topic 75d158a6: EOF [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:45.699] Can't stop node                          [33mid[0m=3a9a3b38d0dcd484 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:14:45.699] caught subscription error in pss sub topic 75d158a6: EOF [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:45.700] Can't stop node                          [33mid[0m=769c948d795d1319 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:14:45.700] caught subscription error in pss sub topic 75d158a6: EOF [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:45.700] Can't stop node                          [33mid[0m=0160436a4791d77b [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:14:45.700] caught subscription error in pss sub topic 75d158a6: EOF [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:45.700] Can't stop node                          [33mid[0m=55bcbf1392d0cd77 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:14:47.470] caught subscription error in pss sub topic 75d158a6: <nil> [33mcaller[0m=api.go:85
--- FAIL: TestForwardBasic (0.00s)
    forwarding_test.go:313: test [Send direct to known, id: [0]]
        msg address: 19d61bc1..., radius: 256
        peer number 0 [19d61bc1...] received 0 messages
        peer number 23 [21c3eb80...] received 1 messages
[33mWARN [0m[07-11|13:14:48.488] caught subscription error in pss sub topic 7f64614e: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:48.488] caught subscription error in pss sub topic 7f64614e: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:49.505] caught subscription error in pss sub topic 7f64614e: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:49.505] caught subscription error in pss sub topic 7f64614e: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:50.523] caught subscription error in pss sub topic 7f64614e: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:14:50.523] caught subscription error in pss sub topic 7f64614e: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:15:06.527] Can't stop node                          [33mid[0m=b2f3a69d52f98037 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:15:06.528] Can't stop node                          [33mid[0m=ece30c5ce40c2769 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:15:06.529] Can't stop node                          [33mid[0m=304c18a9ab1f0bc4 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:15:06.530] Can't stop node                          [33mid[0m=c7c080ce0b5ec6ed [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:15:06.530] Can't stop node                          [33mid[0m=0ffafe08e99119b2 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:15:06.531] Can't stop node                          [33mid[0m=9ed40b70e4c72997 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:15:06.531] Can't stop node                          [33mid[0m=d46598bfcf4437f8 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:15:06.531] Can't stop node                          [33mid[0m=6a517303172afeb0 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:15:06.532] Can't stop node                          [33mid[0m=689c9a0f57a83846 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:15:06.532] Can't stop node                          [33mid[0m=72d3d1448858e393 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:15:06.533] Can't stop node                          [33mid[0m=348e9351b10625a6 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:15:06.533] Can't stop node                          [33mid[0m=3a9a3b38d0dcd484 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:15:06.533] Can't stop node                          [33mid[0m=769c948d795d1319 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:15:06.534] Can't stop node                          [33mid[0m=0160436a4791d77b [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:15:06.534] Can't stop node                          [33mid[0m=55bcbf1392d0cd77 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:15:18.545] noproxhandler                            [33mcaller[0m=pss.go:492
[33mWARN [0m[07-11|13:15:18.545] noproxhandler                            [33mcaller[0m=pss.go:492
[33mWARN [0m[07-11|13:15:18.545] noproxhandler                            [33mcaller[0m=pss.go:492
[31mERROR[0m[07-11|13:15:18.549] peer doesn't have matching pss capabilities, skipping [31mpeer[0m=0x434cd4441857aedb679dc2c4a2d798b567fd73c3a66cbdf53c76135500125c7a [31mcaps[0m=[pss/0]   [31mcaller[0m=pss.go:689
[31mERROR[0m[07-11|13:15:18.549] peer doesn't have matching pss capabilities, skipping [31mpeer[0m=0x6cec09f0b97933201b9c9199238dd4fffd053356bbff8b65f05ce92625f241f1 [31mcaps[0m=[nopss/1] [31mcaller[0m=pss.go:689
[31mERROR[0m[07-11|13:15:18.549] peer doesn't have matching pss capabilities, skipping [31mpeer[0m=0x434cd4441857aedb679dc2c4a2d798b567fd73c3a66cbdf53c76135500125c7a [31mcaps[0m=[pss/0]   [31mcaller[0m=pss.go:689
[31mERROR[0m[07-11|13:15:18.549] peer doesn't have matching pss capabilities, skipping [31mpeer[0m=0x6cec09f0b97933201b9c9199238dd4fffd053356bbff8b65f05ce92625f241f1 [31mcaps[0m=[nopss/1] [31mcaller[0m=pss.go:689
[33mWARN [0m[07-11|13:15:18.549] norawhandler                             [33mcaller[0m=pss.go:488
[33mWARN [0m[07-11|13:15:18.555] Served pss_sendRaw                       [33mreqid[0m=3 [33mt[0m=11.353¬µs [33merr[0m="invalid message length" [33mcaller[0m=handler.go:300
[33mWARN [0m[07-11|13:15:18.555] Served pss_sendRaw                       [33mreqid[0m=4 [33mt[0m=17.217¬µs [33merr[0m="address too long"       [33mcaller[0m=handler.go:300
[33mWARN [0m[07-11|13:15:19.065] caught subscription error in pss sub topic deadbeef: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:15:19.065] caught subscription error in pss sub topic deadbeef: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:15:19.582] caught subscription error in pss sub topic deadbeef: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:15:19.582] caught subscription error in pss sub topic deadbeef: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:15:20.094] caught subscription error in pss sub topic deadbeef: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:15:20.094] caught subscription error in pss sub topic deadbeef: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:15:20.608] caught subscription error in pss sub topic 75d158a6: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:15:20.608] caught subscription error in pss sub topic 75d158a6: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:15:21.123] caught subscription error in pss sub topic 75d158a6: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:15:21.123] caught subscription error in pss sub topic 75d158a6: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:15:21.634] caught subscription error in pss sub topic 75d158a6: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:15:21.634] caught subscription error in pss sub topic 75d158a6: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:15:22.403] caught subscription error in pss sub topic 75d158a6: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:15:22.403] caught subscription error in pss sub topic 75d158a6: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:15:23.168] caught subscription error in pss sub topic 75d158a6: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:15:23.168] caught subscription error in pss sub topic 75d158a6: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:15:23.930] caught subscription error in pss sub topic 75d158a6: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:15:23.930] caught subscription error in pss sub topic 75d158a6: <nil> [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:15:27.139] caught subscription error in pss sub topic 75d158a6: EOF [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:15:27.140] caught subscription error in pss sub topic 75d158a6: EOF [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:15:27.141] Can't stop node                          [33mid[0m=b2f3a69d52f98037 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:15:27.141] caught subscription error in pss sub topic 75d158a6: EOF [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:15:27.141] Can't stop node                          [33mid[0m=ece30c5ce40c2769 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:15:27.141] caught subscription error in pss sub topic 75d158a6: EOF [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:15:27.141] Can't stop node                          [33mid[0m=304c18a9ab1f0bc4 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:15:27.141] caught subscription error in pss sub topic 75d158a6: EOF [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:15:27.142] Can't stop node                          [33mid[0m=c7c080ce0b5ec6ed [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:15:27.142] caught subscription error in pss sub topic 75d158a6: EOF [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:15:27.142] Can't stop node                          [33mid[0m=0ffafe08e99119b2 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:15:27.142] caught subscription error in pss sub topic 75d158a6: EOF [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:15:27.142] Can't stop node                          [33mid[0m=9ed40b70e4c72997 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:15:27.142] caught subscription error in pss sub topic 75d158a6: EOF [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:15:27.143] Can't stop node                          [33mid[0m=d46598bfcf4437f8 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:15:27.143] caught subscription error in pss sub topic 75d158a6: EOF [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:15:27.143] Can't stop node                          [33mid[0m=6a517303172afeb0 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:15:27.143] caught subscription error in pss sub topic 75d158a6: EOF [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:15:27.143] Can't stop node                          [33mid[0m=689c9a0f57a83846 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:15:27.143] caught subscription error in pss sub topic 75d158a6: EOF [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:15:27.143] Can't stop node                          [33mid[0m=72d3d1448858e393 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:15:27.144] caught subscription error in pss sub topic 75d158a6: EOF [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:15:27.144] Can't stop node                          [33mid[0m=348e9351b10625a6 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:15:27.144] caught subscription error in pss sub topic 75d158a6: EOF [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:15:27.144] Can't stop node                          [33mid[0m=3a9a3b38d0dcd484 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:15:27.144] caught subscription error in pss sub topic 75d158a6: EOF [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:15:27.144] Can't stop node                          [33mid[0m=769c948d795d1319 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:15:27.144] caught subscription error in pss sub topic 75d158a6: EOF [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:15:27.145] Can't stop node                          [33mid[0m=0160436a4791d77b [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
[33mWARN [0m[07-11|13:15:27.145] caught subscription error in pss sub topic 75d158a6: EOF [33mcaller[0m=api.go:85
[33mWARN [0m[07-11|13:15:27.145] Can't stop node                          [33mid[0m=55bcbf1392d0cd77 [33merr[0m="server: <nil>, services: map[*network.Bzz:could not save peers to persistence store: could not save peers: leveldb: closed]" [33mcaller[0m=network.go:598
FAIL
[33mWARN [0m[07-11|13:15:28.911] caught subscription error in pss sub topic 75d158a6: <nil> [33mcaller[0m=api.go:85
FAIL	github.com/ethersphere/swarm/pss	82.952s
```

--- a/pss/forwarding_test.go
+++ b/pss/forwarding_test.go
@@ -26,13 +26,11 @@ type testCase struct {
 	errors    string
 }
 
-var testCases []testCase
-
 // the purpose of this test is to see that pss.forward() function correctly
 // selects the peers for message forwarding, depending on the message address
 // and kademlia constellation.
 func TestForwardBasic(t *testing.T) {
-	t.Skip("Flaky on macOS on local machines")
+	var testCases []testCase
 	baseAddrBytes := make([]byte, 32)
 	for i := 0; i < len(baseAddrBytes); i++ {
 		baseAddrBytes[i] = 0xFF

